tool/site/client-react/src/components/my-prescriptions/myPrescriptions.tsx

```diff
@@ -10,22 +10,20 @@ interface PrescriptionSummary {
   id: string
 }
 
-export const MyPrescriptions : React.FC<Prescriptions> = (
-  prescriptions
-) => {
+export const MyPrescriptions : React.FC<Prescriptions> = prescriptions => {
   return (
     <>
       <PrescriptionGroupTable
         name="Sent Prescriptions"
         description="Prescriptions ready to release"
         prescriptions={prescriptions.sentPrescriptions}
-        actions={{ view: true, release: true }}
+        actions={{view: true, release: true}}
       />
       <PrescriptionGroupTable
         name="Released Prescriptions"
         description="Prescriptions ready to dispense"
         prescriptions={prescriptions.releasedPrescriptions}
-        actions={{ view: true, releaseReturn: true, dispense: true }}
+        actions={{view: true, releaseReturn: true, withdraw: true, dispense: true}}
```

Comment:
The withdraw action would make sense in the context of a "Dispensed Prescriptions / Prescriptions being dispensed" group table, released prescriptions are before any dispense interactions so there's nothing to peform withdraw on

Reply:
I've clearly misunderstood this part, I'll add a new group table and shuffle the backend MyPrescriptions logic